.github/workflows/ci.yaml

```diff
@@ -1,10 +1,17 @@
 name: Continuous Integration
 
-on: [push, pull_request]
+on:
+  push:
+  pull_request:
+  schedule:
+    - cron: '0 0 */90 * *'
+
 
 jobs:
   build:
     runs-on: self-hosted
+    env:
+      DEPOT_TOOLS_UPDATE: 0
```

Comment:
why this env is neccessary?

Reply:
It's not necessary. Disable update for saving time. Let me remove it.